### PR TITLE
fix (Add all components): adding components on homepage

### PR DESCRIPTION
**Pull request by andrii1**
# Description

Please provide a short summary explaining what this PR is about.

Fixes # (issue)

# How to test?

Please provide a short summary how your changes can be tested?

# Checklist

- [ ] I have performed a self-review of my own code
- [ ] I have followed the name conventions for CSS Classnames and filenames, Components names and filenames, Style filenames, if you are in doubt check the the project README.MD and here https://github.com/HackYourFuture-CPH/curriculum/blob/master/review/review-checklist.md
- [ ] I have commented my code, particularly in hard-to-understand areas, if you code was simple enough mark the box anyway
- [ ] I have made corresponding changes to the documentation, if you code was simple enough mark the box anyway
- [ ] This PR is ready to be merged and not breaking any other functionality

### CODE REVIEW

File: packages/client/package.json [hunk 1]
```diff
@@ -3,9 +3,15 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@fortawesome/fontawesome-svg-core": "^6.2.1",
+    "@fortawesome/free-brands-svg-icons": "^6.2.1",
+    "@fortawesome/free-regular-svg-icons": "^6.2.1",
+    "@fortawesome/free-solid-svg-icons": "^6.2.1",
+    "@fortawesome/react-fontawesome": "^0.2.0",
     "@testing-library/jest-dom": "^5.16.4",
```
Comment (shpomp) [inline packages/client/package.json:11]: No changes to the dependencies that are not agreed upon with everybody. 
Icons are in the design system, and should be placed in /public and used from there. 
Fix that, then I can review the rest. 🙏🏽

**Review by shpomp — APPROVED** [review]

**Review by shpomp — CHANGES_REQUESTED** [review]
-

**Review by Hailegebriel22 — APPROVED** [review]

File: packages/client/package.json (hunk 1 above)
Comment (andrii1) [reply packages/client/package.json:11]: ok, will remove fontawesome; but maybe we can keep that as I already did it?

Comment (shpomp) [reply packages/client/package.json:11]: no 😅 
sorry 🙏🏽

**Comment by shpomp** [comment]
Merging with an exception. Agreed on no more monster PRs in the future.

**Comment by shpomp** [comment]
@Hailegebriel22 @Hanouj  Make sure to familiarise yourselves with all of the code 👌🏽

**Pull request merged**